deployment/terraform/modules/rds/main.tf

```diff
@@ -45,3 +45,20 @@ resource "aws_db_instance" "default" {
     ignore_changes = ["final_snapshot_identifier"]
   }
 }
+
+resource "aws_cloudwatch_metric_alarm" "default_rds_cpu_utilization" {
+  alarm_name          = "${var.resource_name_prefix}-default-rds-cpu-utilization"
+  comparison_operator = "GreaterThanOrEqualToThreshold"
+  evaluation_periods  = "5"
+  metric_name         = "CPUUtilization"
+  namespace           = "AWS/RDS"
+  period              = "300"
+  statistic           = "Average"
+  threshold           = "25"
+  alarm_description   = "Monitors CPU utilization of default RDS instance"
```

Comment:
`"Monitors CPU utilization of ${aws_db_instance.default.id}"` or whatever is the human readable name.

Reply:
I think the most human readable name is "default."  There are no tags or the like for RDS.  We can change that though.